docs/reference-guide/index.md

```diff
@@ -0,0 +1,27 @@
+---
+myst:
+  html_meta:
+    "description": "Plone Reference guide"
+    "property=og:description": "Plone Reference guide"
+    "property=og:title": "Reference guide"
+    "keywords": "Plone 6, reference guide"
+---
+
+# Reference guide
+
+This reference guide details functions, modules, and objects included in Plone, describing what they are and what they do. For learning how to use Plone, see the [complete documentation](https://6.docs.plone.org/index.html).
+
+```{note}
+This part of the documentation is under construction.
+You can find documentation for plone in various locations through the [search feature](https://6.docs.plone.org/search.html?q=plone).
+You can help consolidate all of reference material here, even if it is to let us know what is missing, by [creating an issue](https://github.com/plone/documentation/issues/new?assignees=&labels=&projects=&template=new-issue-form.yml).
+```
+
+## Plone REST API
+
+[Plone REST API](https://6.docs.plone.org/plone.restapi/docs/source/index.html)
+
+
+## Plone.api
+
+[Plone.api](https://6.docs.plone.org/plone.api/index.html)
```

Comment:
I think we can put the above two headings and links into a toctree. When we put things into toctrees in different files, it destroys the site navigation, and they already exist elsewhere in the submodules.

However, I've used `sphinx-reredirects` with landing pages to trick it. See for example the source of [Contributing to plone.api](https://6.docs.plone.org/contributing/plone-api.html) and other files in the contributing directory. If you'd like to try hacking that out, please be my guest!